pyaedt/hfss.py

```diff
@@ -608,58 +607,72 @@ def _create_waveport_driven(
-        return self._create_boundary(portname, props, "Wave Port")
-
-    @pyaedt_function_handler()
+        return self._create_boundary(port_name, props, "Wave Port")
+
+    @pyaedt_function_handler(
+        obj="assignment",
+        mat="material",
+        cond="conductivity",
+        perm="permittivity",
+        usethickness="use_thickness",
+        isinfgnd="is_infinite_ground",
+        istwoside="is_two_side",
+        isInternal="is_internal",
+        issheelElement="is_shell_element",
+        usehuray="use_huray",
+    )
     def assign_coating(
         self,
-        obj,
-        mat=None,
-        cond=58000000,
-        perm=1,
-        usethickness=False,
+        assignment,
+        material=None,
+        conductivity=58000000,
+        permittivity=1,
+        use_thickness=False,
         thickness="0.1mm",
         roughness="0um",
-        isinfgnd=False,
-        istwoside=False,
-        isInternal=True,
-        issheelElement=False,
-        usehuray=False,
+        is_infinite_ground=False,
+        is_two_side=False,
+        is_internal=True,
+        is_shell_element=False,
+        use_huray=False,
         radius="0.5um",
         ratio="2.9",
+        name=None,
     ):
         """Assign finite conductivity to one or more objects or faces of a given material.
 
         Parameters
         ----------
-        obj : str or list
+        assignment : str or list
             One or more objects or faces to assign finite conductivity to.
-        mat : str, optional
+        material : str, optional
             Material to use. The default is ``None``.
-        cond : float, optional
+        conductivity : float, optional
             If no material is provided, a conductivity value must be supplied. The default is ``58000000``.
-        perm : float, optional
+        permittivity : float, optional
             If no material is provided, a permittivity value must be supplied. The default is ``1``.
-        usethickness : bool, optional
+        use_thickness : bool, optional
             Whether to use thickness. The default is ``False``.
         thickness : str, optional
             Thickness value if ``usethickness=True``. The default is ``"0.1mm"``.
         roughness : str, optional
             Roughness value  with units. The default is ``"0um"``.
-        isinfgnd : bool, optional
+        is_infinite_ground : bool, optional
             Whether the finite conductivity is an infinite ground. The default is ``False``.
-        istwoside : bool, optional
+        is_two_side : bool, optional
             Whether the finite conductivity is two-sided. The default is ``False``.
-        isInternal : bool, optional
+        is_internal : bool, optional
             Whether the finite conductivity is internal. The default is ``True``.
-        issheelElement : bool, optional
+        is_shell_element : bool, optional
             The default is ``False``.
```

Comment:
```suggestion
            Whether the finite conductivity is a shell element.
            The default is ``False``.
```
Brief description was missing. Verify the one suggested.